graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlWSSubProtocol.java

```diff
@@ -352,7 +353,9 @@ private static void writeNext(WebSocketWriter out, String operationId, Execution
                 "payload", executionResult.toSpecification());
         final String event = serializeToJson(response);
         logger.trace("NEXT: {}", event);
-        out.tryWrite(event);
+        if (!out.tryWrite(event)) {
+            throw ClosedStreamException.get();
```

Comment:
`.write()` will raise `ClosedStreamException` if it is closed.
```suggestion
        out.write(event);
```
Should we also use `write()` in other places to cancel the publisher?
A considering point is that `writeError()` in `completeWithError()` may cause an error again because `out` was closed already. If `writeNext()` fails, we may skip `writeError()`.

Reply:
> Should we also use `write()` in other places to cancel the publisher?
I think `writeErrors` should be considered to use `write`  instead of `tryWrite`.
https://github.com/line/armeria/blob/89c7cf4314181ca7322fb6685becc9ffd3570f5a/graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlWSSubProtocol.java#L358

> A considering point is that `writeError()` in `completeWithError()` may cause an error again because `out` was closed already. If `writeNext()` fails, we may skip `writeError()`.
I think we can fix like this below
```java
private static void writeError(WebSocketWriter out, String operationId, Throwable t) {
        ...
        try {
            final String event = serializeToJson(errorResponse);
            logger.trace("ERROR: {}", event);
            out.write(event);
        } catch (Throwable e) {
            logger.warn("Error serializing error event", e);
            if (out.isOpen()) {
                out.close(e);
            }
        }
    }
```

Reply:
We may simply return in the begging of the method if `t` is an instance of `ClosedStreamException`